Sources/StreamChat/Deprecations.swift

```diff
@@ -23,6 +23,14 @@ public extension _ChatChannel {
     var watchers: Set<_ChatUser<ExtraData.User>> { Set(lastActiveWatchers) }
 }
 
+public extension _ChatMessage {
+    /// Quoted message id.
+    ///
+    /// If message is inline reply this property will contain id of the message quoted by this reply.
+    @available(*, deprecated, renamed: "quotedMessage.id")
+    var quotedMessageId: MessageId? { quotedMessage?.id }
```

Comment:
Did you test if this works properly? Shouldn't the suggestion be `quotedMessage?.id ` such that it compilers right away?

Reply:
The `renamed: "quotedMessage?.id"` doesn't compile 🤷‍♂️ Replaced it with `message: "Use quotedMessage?.id instead"`